### PR TITLE
Feature/show general kyc details

### PR DESCRIPTION
**Pull request by Datikzz**

### CODE REVIEW

File: src/components/User/Users/components/UserDetails/UserDetails.AccreditedKycViewer.vue [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<template>
+  <div class="user-details-accredited-kyc-viewer">
+    <h2>Current user is applyied for Accredited KYC</h2>
```
Comment (quinn71) [inline src/components/User/Users/components/UserDetails/UserDetails.AccreditedKycViewer.vue]: typo

Comment (quinn71) [reply src/components/User/Users/components/UserDetails/UserDetails.AccreditedKycViewer.vue]: Add "Please ensure he has a valid proof document"

File: src/components/User/Users/components/UserDetails/UserDetails.AccreditedKycViewer.vue [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+<script>
+import KycGeneralSection from './UserDetails.Kyc'
+import UserDocGetter from '@comcom/getters'
+export default {
+  components: {
+    KycGeneralSection,
+    UserDocGetter
+  },
+  props: ['kyc', 'user', 'isKycLoaded', 'isKycLoadFailed']
```
Comment (quinn71) [inline src/components/User/Users/components/UserDetails/UserDetails.AccreditedKycViewer.vue]: just don't render the data component if loading failed

File: src/components/User/Users/components/UserDetails/UserDetails.Kyc.vue [hunk 3]
```diff
@@ -4,46 +4,14 @@
     <div class="user-details-account__info">
       <p>Can be uploaded one of the documents:</p>
       <p> - Valid passport;</p>
-      <p>- Driving license - full, not provisional, with a color photograph;</p>
-      <p> - Valid National ID card;
-        <p />
-        <p>- In all cases handwritten documents or those in a foreign script that cannot be read, i.e Arabic or Cyrillic
-          will not be accepted unless translated and notarized.</p>
+      <p> - Valid identity card;</p>
```
Comment (quinn71) [inline src/components/User/Users/components/UserDetails/UserDetails.Kyc.vue]: inconsistent

File: src/components/User/Users/components/UserDetails/UserDetails.Kyc.vue [hunk 4]
```diff
@@ -4,46 +4,14 @@
...
+      <p> - Valid Residence Permit;</p>
+      <p> - Driving license - full, not provisional, with a color photograph;</p>
+      <p> - In all cases handwritten documents or those in a foreign script that cannot be read, i.e Arabic or Cyrillic
```
Comment (quinn71) [inline src/components/User/Users/components/UserDetails/UserDetails.Kyc.vue]: remove this paragraph

File: src/components/User/Users/components/UserDetails/UserDetails.vue [hunk 5]
```diff
@@ -49,12 +49,24 @@
...
+              :isKycLoaded="isKycLoaded"
+              :isKycLoadFailed ="isKycLoadFailed" />
+            <verified-kyc-viewer
+              v-if="requestToReview.accountRoleToSet === ACCOUNT_ROLES.us_verified"
+              :kyc="kyc"
+              :user="user"
+              :isKycLoaded="isKycLoaded"
+              :isKycLoadFailed ="isKycLoadFailed" />
+            <accredited-kyc-viewer
+              v-if="requestToReview.accountRoleToSet === ACCOUNT_ROLES.us_accredited"
```
Comment (quinn71) [inline src/components/User/Users/components/UserDetails/UserDetails.vue]: why snack case

File: src/config.js [hunk 6]
```diff
@@ -40,7 +40,9 @@ export default Object.assign(
       notVerified: '',
       general: '',
       corporate: '',
-      blocked: ''
+      blocked: '',
+      us_accredited: '',
```
Comment (quinn71) [inline src/config.js]: usAccredited

File: src/components/User/Users/components/UserDetails/UserDetails.AccreditedKycViewer.vue (hunk 1 above)
Comment (Datikzz) [reply src/components/User/Users/components/UserDetails/UserDetails.AccreditedKycViewer.vue]: edited

File: src/components/User/Users/components/UserDetails/UserDetails.AccreditedKycViewer.vue (hunk 2 above)
Comment (Datikzz) [reply src/components/User/Users/components/UserDetails/UserDetails.AccreditedKycViewer.vue]: edited

File: src/components/User/Users/components/UserDetails/UserDetails.Kyc.vue (hunk 3 above)
Comment (Datikzz) [reply src/components/User/Users/components/UserDetails/UserDetails.Kyc.vue]: edited

File: src/components/User/Users/components/UserDetails/UserDetails.Kyc.vue (hunk 4 above)
Comment (Datikzz) [reply src/components/User/Users/components/UserDetails/UserDetails.Kyc.vue]: removed

File: src/config.js (hunk 6 above)
Comment (Datikzz) [reply src/config.js]: edited

File: src/components/User/Users/components/UserDetails/UserDetails.vue (hunk 5 above)
Comment (Datikzz) [reply src/components/User/Users/components/UserDetails/UserDetails.vue]: edited

**Pull request closed without merging**